src/CONST.ts

```diff
@@ -875,7 +875,8 @@ const CONST = {
     },
     TIMING: {
         CALCULATE_MOST_RECENT_LAST_MODIFIED_ACTION: 'calc_most_recent_last_modified_action',
-        SEARCH_RENDER: 'search_render',
+        // ASK: Should we also update this name?
+        CHAT_FINDER_RENDER: 'chat_finder',
```

Comment:
Hmm it seems like we save this metric to Firebase and Grafana. So Maybe we should just rename the key, but leave the value as is.
```suggestion
        CHAT_FINDER_RENDER: 'search_render',
```